src/main/java/racingcar/controller/Controller.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.controller;
+
+import racingcar.model.RandomNumberGenerator;
+import racingcar.view.validator.InputValidator;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+import racingcar.model.Car;
+import racingcar.model.Referee;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Controller {
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+    private final InputValidator inputValidator = new InputValidator();
+    private List<Car> cars;
+    private final Referee referee = new Referee();
+
+    public void run() {
+        List<String> carNames = inputView.getCarNames();
+        inputValidator.validateAvailableCarNames(carNames);
+        int tryNumber = inputView.getTryNumber();
+        generateCarList(carNames);
+        outputView.printResultHeaderPrompt();
+        while(tryNumber-- > 0){
+            runRace();
+        }
+        outputView.printWinners(referee.generateWinners(cars));
+    }
+
+    private void runRace() {
+        for(Car car : cars) {
+            car.move();
+            outputView.printResult(car.getName(), car.getPosition());
+        }
+        OutputView.parseLine();
+    }
+
+    private void generateCarList(List<String> carNames) {
+        cars = new ArrayList<>();
+        for(String carName : carNames){
```

Comment:
```suggestion
        for(String carName : carNames) {
```
다른 파일들도 그렇고 전반적으로 formatting 적용이 필요한 곳이 보이네요.
IntelliJ에서 기본적으로 자동 정렬을 제공해줍니다! macOS: `Cmd + Alt + L`, windows: `Ctrl + Alt + L`
커밋 전에 한번씩 사용해보는 습관을 들여보아요!
[save마다 정렬](https://hianna.tistory.com/791)이 되도록 설정할 수도 있습니다!